tests/test_axon_point.py

```diff
@@ -1,15 +1,14 @@
-import os
-
+from pathlib import Path
 from morphio import Morphology
 from nose.tools import eq_
 from morph_tool import axon_point_section
 
 
-DATA = os.path.join(os.path.abspath(os.path.dirname(__file__)), "data")
+DATA = Path(__file__).absolute().parents[0] / "data"
```

Comment:
why `absolute` and why not just `parent` https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.parent?

Reply:
```suggestion
DATA = Path(__file__).absolute().parent / "data"
```

Reply:
this discussion have not been resolved

Reply:
done (notice all other tests don't use pathlib)